modules/services/cloudwatch/ec2-monitoring.tf

```diff
@@ -0,0 +1,22 @@
+## Default EC2 monitoring metrics
+
+resource "aws_cloudwatch_metric_alarm" "high_cpu_alert_ec2" {
+  count             = length(var.ec2_instances)
+  alarm_name        = "High CPU Utilization - EC2 Instance ${element(var.ec2_instances, count.index)}-${var.customer_name}-${var.environment}"
+  alarm_description = "CPU utilization exceeded acceptable threshold"
+
+  actions_enabled = var.default_ec2_alarms_enabled
+  alarm_actions   = [aws_sns_topic.high_priority_alert.arn]
+  ok_actions      = [aws_sns_topic.high_priority_alert.arn]
+
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = "12"
+  threshold           = "90"
+  metric_name         = "CPUUtilization"
```

Comment:
The intension here is to eventually have predefined metrics for a whole array of services, like mongo, sql, ec2, elastic search, etc. and each service having a multiple _standard_  metrics. 

Making each and every part of the metrics configurable, would not only result in having lots and lots of variables, it would also defeate the purpose of having this module, if anyone using it would anyway have to pass everything to it, they might as well create their own module.

These metrics are therefore by-default set as disabled. Anyone using this module can either explictly enable the default metrics for some of the the services, or define their own custom metrics.

Currently there are only two standard metrics defined, as I lacked reference materials for appropriate metrics to be set as standard. If you have some reference materials, or reference implementations, please point those out, so I can configure them here.